source/features/repo-age.tsx

```diff
@@ -123,8 +123,9 @@ void features.add({
 		pageDetect.isRepoRoot
 	],
 	exclude: [
-		pageDetect.isEmptyRepoRoot
+		// Excludes empty repos, compatible with `awaitDomReady: false`
+		() => !select.exists('link[rel="canonical"]')
```

Comment:
I think this should be merged back as `isEmptyRepoRoot`

```js
// On empty repos, there's only isRepoHome
const isEmptyRepoRoot = () => isRepoHome() && !select.exists('link[rel="canonical"]');
```

Reply:
In the future, changes like this one should be made on `github-url-detection`, not here.

Reply:
That was my intention. I brought it up here to explain the issue.

I will make the 2 pr's now.


In the future would you like me to create an issue there first. From my experience it usually works out best, when we discuss it here and then send the pr.

Reply:
Ok